src/monkey-patch.android.ts

```diff
@@ -1,39 +1,40 @@
-// @ts-expect-error because there is no corresponding type definition
```

Comment:
I think this line is needed - otherwise CI is failing 😔

Reply:
added this